db/migrate/20221013213434_create_notifications.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateNotifications < ActiveRecord::Migration[7.0]
+  def change
+    create_table :notifications do |t|
+      t.references :recipient, null: false
+      t.string :type, null: false
```

Comment:
I think we needed this column because `recipient` was a polymorphic relation. Do we still need it?

Reply:
`recipient` stores the user who receives the notification. It is not a polymorphic relation anymore. It refers to the user model. @JulianPasquale

Reply:
Yeah I understand that, but my question was not for the `recipient` column, but for the `type` column. Why do we need that one?